scripts/did_finder.py

```diff
@@ -120,19 +165,18 @@ def init_rabbit_mq(rabbitmq_url, retries, retry_interval):
                 raise eek
 
 
-# Main Script
-args = parser.parse_args()
-
-site = args.site
-prefix = args.prefix
-threads = args.threads
-print("--------- ServiceX DID Finder ----------------")
-print("Threads: " + str(threads))
-print("Site: " + str(site))
-print("Prefix: " + str(prefix))
+if __name__ == "__main__":
+    # Main Script
+    logger = initialize_logging()
+    args = parser.parse_args()
 
-did_client = DIDClient()
-replica_client = ReplicaClient()
-rucio_adapter = RucioAdapter(did_client, replica_client)
+    site = args.site
+    prefix = args.prefix
+    threads = args.threads
+    logger.info("None ServiceX DID Finder starting up: " +
```

Comment:
What is the purpose of `"None"` here?

Reply:
That was something left over from a prior version when I didn't have the request id handling dialed in.